test/nn/test_pooling.py

```diff
@@ -708,17 +708,59 @@ def check(x, args, expected):
             self.assertEqual(model(x), expected)
 
         # Pooling args: (kernel_size, stride, padding, dilation, return_indices, ceil_mode)
-        check([[]], (1, None, 0, 1, False, False), [[]])
-        check([[[]]], (1, None, 0, 1, False, False), [[[]]])
-        check([[[]]], (2, 1, 1, 2, False, True), [[[]]])
```

Comment:
These are not supported now even for `requires_grad=False` (BC-breaking).

Reply:
I'm not sure if this qualifies as a BC-breaking change since bug fixes are, by definition, not BC-breaking.

How redundant is this test with ErrorInputs and SampleInputs? Is there some logic in it we should port to those? Can we delete this test?

Reply:
Cool will remove the BC-breaking label.

The failures have been moved the error inputs (which is sufficient). Thanks!